mfr/server/handlers/core.py

```diff
@@ -104,6 +104,8 @@ def NAME(self):
 
         try:
             self.url = self.request.query_arguments['url'][0].decode('utf-8')
+            # TODO should probably use furl here, but maybe wait till after furl is upgraded
+            self.url += '&mfr=true'
```

Comment:
This is fine since `self.url` is a literal string instead of an object as long as the URL does not contain fragment parameters.